ol/types/src/config.rs

```diff
@@ -61,10 +61,23 @@ pub struct AppCfg {
 }
 
 /// Get a AppCfg object from toml file
-pub fn parse_toml(path: PathBuf) -> Result<AppCfg, Error> {
+pub fn parse_toml(path: Option<PathBuf>) -> Result<AppCfg, Error> {
     // let mut config_toml = path.to_str().unwrap().to_owned()).expect("could not parse app config from file");
+   
+    let cfg_path = match path {
```

Comment:
.unwrap_or(dirs::home_dir().unwrap().join(".0L").join("0L.toml"));

// this unwrap is likely safe in this case. nothing to recover from.